src/commands/deletion_management.py

```diff
@@ -0,0 +1,94 @@
+"""
+deletion_management.py - commands for managing deletion marking on cases.
+
+Copyright (c) 2018 The Fuel Rat Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+
+import asyncio
+import datetime
+import re
+import uuid
+from datetime import timezone
+from typing import Optional
+
+import humanfriendly
+
+from src.packages.commands import command
+from src.packages.context.context import Context
+from src.packages.epic import Epic
+from src.packages.permissions.permissions import require_permission, RAT, OVERSEER, require_channel
+from src.packages.mark_for_deletion import MarkForDeletion
+from src.packages.quotation.rat_quotation import Quotation
+from src.packages.rat import Rat
+from src.packages.rescue import Rescue
+from src.packages.utils import Platforms, Status
+
+
+# User input validation helper
+def _validate(ctx: Context, validate: str) -> Optional[Rescue]:
```

Comment:
[Suggestion] Duplicate from `case_management.py`. Maybe a method in `RatBoard` instead?